generator/generator_test.go

```diff
@@ -421,6 +422,53 @@ func TestProcessAdditionWithDifferentPartClass(t *testing.T) {
 	}
 
 }
+
+func TestProcessSetParam(t *testing.T) {
```

Comment:
@minhaj10p can we write a negative test case as well to make sure that if a parameter isn't found in a prose, it shouldn't be changed after applying a profile?

Reply:
adding negative test